01_05_Conditions.txt

```diff
@@ -4,92 +4,170 @@ Conditions
 
 == Bleeding ==
 
-    d10/2+1 (drop the remainder) stackable damage per turn. Broken with 
-Medicine Skill check. DC varies. If you have a bleed condition, seek medical aid
-immediately, You CAN Bleed Out, you have been warned.
+    Bleeding is when a character has sustained damage to the systems of their
+anatomy that contian liquids vital to their survival. This can happen to both
+biologic targets, who usually leak blood, and to cybernetic targets, who can 
+leak coolant and lubricant.
+
+    A Character that is bleeding takes d10/2+1 (drop the remainder) stackable 
+damage once per round at the beginning of their turn. The bleeding can be 
+staunched with Medicine Skill check. The Medicine (Dex) DC to stop varies; the
+cause of the bleed effect should list it in the effect description.
 
 == Burning ==
 
-    2d10 damage per turn. Broken with a Maneuver Skill Check. DC 40. 
-Every 2 turns, the degree of the burn intensifies
+    A character that is on fire, or has glowing metal on them (such as white
+phosphorous) is burning. Burning is different than a Plasma Burn. For Plasma
+Burns, see the "Plasma Burn" Condition further below.
+
+    A Burning character takes 2d10 damage per stack of burn per round at the 
```

Comment:
While we're stocklpiling little edits, this damage needs to be typed to Fire (or thermal).